semantic_release/vcs_helpers.py

```diff
@@ -6,9 +6,9 @@
 from .settings import config
 
 try:
-    repo = Repo('.git', search_parent_directories=True)
+    repo = Repo('.', search_parent_directories=True)
 except NoSuchPathError:
-    repo = None
+    raise RuntimeError("Not in a valid git repository")
```

Comment:
This will raise the error on import, we rather want to throw when a function is called since it is then possible to catch it and show a nice error message.